authentication-guides/workday/README.md

```diff
@@ -107,41 +107,41 @@ Run the **`Activate Pending Security Policy Changes`** task to activate permissi
 
 Universal search for **`Register API Client for Integrations`**
 
-![https://developer.moveworks.com/static/3f4721c823875fbc16322fb8cc12f78c/410f3/Untitled_10.png](https://developer.moveworks.com/static/3f4721c823875fbc16322fb8cc12f78c/410f3/Untitled_10.png)
+![search for register api client for integrations](https://developer.moveworks.com/static/3f4721c823875fbc16322fb8cc12f78c/410f3/Untitled_10.png)
```

Comment:
This was moved to the help docs site.

Reply:
https://help.moveworks.com/docs/workday-access-requirements

Reply:
brought image locally